yarn.lock

```diff
@@ -8067,7 +8066,6 @@ npm@^8.3.0:
     "@npmcli/fs" "^2.1.0"
     "@npmcli/map-workspaces" "^2.0.3"
     "@npmcli/package-json" "^2.0.0"
-    "@npmcli/promise-spawn" "^3.0.0"
     "@npmcli/run-script" "^4.2.1"
```

Comment:
@viceice @rarkins can you explain this? i ran yarn install and npm 8.19.2 is missing 3 dependencies? is that normal?